components/camel-rabbitmq/src/main/java/org/apache/camel/component/rabbitmq/RabbitMQComponent.java

```diff
@@ -185,6 +185,8 @@ protected RabbitMQEndpoint createEndpoint(String uri,
             }
         }
 
+        @SuppressWarnings("unchecked")
+        Map<String, Object> args = resolveAndRemoveReferenceParameter(params, "args", Map.class, getArgs());
```

Comment:
I think this is not needed, under by `setProperties` is doing the job to resolve the args param

Reply:
Same as my next comment